src/pages/Editor/index.tsx

```diff
@@ -102,6 +102,19 @@ const Editor = () => {
   }, [theme]);
 
   // Functions:
+  const transform = (text: string, id?: string) => {
```

Comment:
Function name is too generic, make it a little more specific. Like, `transformTemplateContent` or `transformAttributesInTemplateContent`. It's better to have a long, descriptive function name than a succinct function name that doesn't tell us anything specific.